frontend/src/components/pages/ImportStoryPage.tsx

```diff
@@ -0,0 +1,184 @@
+import React from "react";
+import Select from "react-select";
+import {
+  Box,
+  Button,
+  Flex,
+  FormControl,
+  FormLabel,
+  Heading,
+  Input,
+  Link,
+  Text,
+  Textarea,
+} from "@chakra-ui/react";
+import { Icon } from "@chakra-ui/icon";
+import { MdArrowBackIosNew, MdFolder } from "react-icons/md";
+import DropdownIndicator from "../utils/DropdownIndicator";
+import Header from "../navigation/Header";
+import { colourStyles } from "../../theme/components/Select";
+
+const ImportStoryPage = () => {
+  return (
+    <Flex direction="column" height="100vh" justifyContent="space-between">
+      <Header />
+      <Flex
+        direction="column"
+        flex="1"
+        padding="40px 120px 40px 120px"
+        overflow="auto"
+      >
+        <Link
+          color="blue.100"
+          fontWeight="bold"
+          href="#/"
```

Comment:
nit: redirect to `#/?tab=1`
this will go to the manage stories tab specifically

Reply:
Should this happen even if the user accesses the import story page from the ManageStoryTranslations page?